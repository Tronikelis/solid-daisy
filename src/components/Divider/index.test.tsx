import { render } from "@solidjs/testing-library";
import { describe, expect, it } from "vitest";

import { Divider } from ".";

describe("<Divider />", () => {
    it("renders correctly", () => {
        const { container } = render(() => <Divider />);
        expect(container).toMatchSnapshot();
    });
});
