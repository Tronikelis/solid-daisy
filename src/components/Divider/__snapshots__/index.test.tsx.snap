// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`<Divider /> > renders correctly 1`] = `
<div>
  <div
    class="divider m-0"
  />
</div>
`;
